ci_build_images/sles.Dockerfile

```diff
@@ -26,11 +30,15 @@ RUN zypper -n update \
     libboost_system1_66_0-devel \
     libcurl-devel \
     libffi-devel \
+    libfmt8 \
     libgnutls-devel \
+    liblz4-devel \
+    libopenssl-3-devel \
+    liburing2-devel \
     perl-Net-SSLeay \
     policycoreutils \
-    python3-devel \
-    python3-pip \
+    python312-devel \
```

Comment:
suggest to go with python311-devel/pip here as that is the longer term supported python 3.x on SLE.

Reply:
Thanks @dirkmueller . The python is supporting the BB application. We've got a range of python bases so its less critical, but good to know.